world-explore/src/app/home/home.page.html

```diff
@@ -23,10 +23,10 @@
       [ngClass]="{ 'favorite': favoriteCountries.has(country.country) }" class="country-item" (ionSwipe)="toggleFavorite(country, $event)">
       <ion-item>
         <ion-thumbnail slot="start">
-          <ng-container *ngIf="getSafeFlagUrl(country.country)">
-            <ion-img [src]="getSafeFlagUrl(country.country)" class="country-flag"></ion-img>
+          <ng-container *ngIf="country.flagUrl">
```

Comment:
@ZiadAmrEmbabi 
plz make sure to apply safe check for all data parsing to prevent duplicate comment 
Ex :  country?.country